test/e2e/framework.go

```diff
@@ -1326,6 +1326,21 @@ func (data *TestData) getAntreaPodOnNode(nodeName string) (podName string, err e
 	return pods.Items[0].Name, nil
 }
 
+// getFlowAggregator retrieves the name of the Flow-Aggregator Pod (flow-aggregator-*) running on a specific Node.
+func (data *TestData) getFlowAggregator(nodeName string) (podName string, err error) {
```

Comment:
please use this function in the implementation of `gracefulExitFlowAggregator` to avoid code duplication

Reply:
Done